.github/workflows/deploy.yml

```diff
@@ -55,3 +55,12 @@ jobs:
       - name: Deploy to GitHub Pages
         id: deployment
         uses: actions/deploy-pages@v2
+
+      # See https://www.algolia.com/doc/rest-api/crawler/#reindex-with-a-crawler
+      - name: Kickoff a crawl
+        run: |
+          curl \
+            -H "Content-Type: application/json" \
+            -X POST \
+            --user ${{ secrets.THEJCANNON_ALGOLIA_CRAWLER_USER_ID }}:${{ secrets.THEJCANNON_ALGOLIA_CRAWLER_API_KEY }} \
```

Comment:
Btw, does someone else have access to this account? E.g. something in the 1password vault that supposedly exists / associated with @WorkerPants's email?

Reply:
I invited Benjy. If you want I can invite you as well.

Inviting worker pants might also work in addition. I'll try it out

Reply:
Did it work?

Reply:
Benjy says he didn't see the email 😓

Reply:
It says invitation was sent. Let's see if Benjy can get logged in as WorkerPants via GitHub SSO and see the app.

Reply:
OK Benjy got logged in, but oddly the crawler(s) seem to be per-user. Will keep banging on it.